test-cases/tests/alerts/c18-validate-api-usage-alerts.md

```diff
@@ -14,148 +14,172 @@ tags:
+
+8. Next, open the 3Scale admin console and login with `admin` as the username and the retrieved password
+
+   ```sh
+   open "https://$(oc get route -n redhat-rhoam-3scale | grep 3scale-admin | awk {'print $2'})"
+   ```
+
+9. Click on `Ok, how does 3scale work?` and follow the 3Scale wizard to create an API
+
+10. Once on your API overview page, click on `Integration` on the left, then on `Configuration`
+
+11. Take note of the `example curl for testing` for `Staging-APIcast`
+
+12. Trigger API Usage alerts
+
+    | Alert to fire                        | `numRequests` | `recipients`          |
+    | ------------------------------------ | ------------- | --------------------- |
+    | RHOAMApiUsageLevel1ThresholdExceeded | 43            | BU and Customers      |
+    | RHOAMApiUsageLevel2ThresholdExceeded | 46            | BU and Customers      |
+    | RHOAMApiUsageLevel3ThresholdExceeded | 51            | BU, Customers and SRE |
+
+
+    Due to the timing of the Prometheus scrape interval, getting alerts to fire on a per 1 minute basis can be challenging. To cater for this timing issue, perform the following steps:
+
+      Prepare the API request command below, replacing `$numrequests` with the count set in the table above. Also make sure to replace `DUMMY_URL` and `DUMMY_KEY` with the values retrieved from the 3Scale console previously
+
+      *NOTE:* Do not run the command yet!
+
+      ```sh
+      for i in {1..$numRequests}; do curl -i https://<DUMMY_URL>//?user_key=<DUMMY_KEY>; done
+      ```
+
+      Open the Prometheus console:
+
+      ```sh
+      open "https://$(oc get routes prometheus-route -n redhat-rhoam-middleware-monitoring-operator -o jsonpath='{.spec.host}')"
+      ```
+
+      Add the following expression into the `Expression` field in the console:
+
+      ```
+      increase(ratelimit_service_rate_limit_apicast_ratelimit_generic_key_slowpath_total_hits[1m])
+      ```
+
+      Click the `Execute` button. A `0` count should be returned.
+
+      *NOTE:* The next steps require preparation and need to be run quickly, please read ahead before running each step
+
+      Next, send a request to the 3Scale, making sure to replace `DUMMY_URL` and `DUMMY_KEY` with the values retrieved from the 3Scale console earlier.
+
+      ```sh
+      curl -i https://<DUMMY_URL>//?user_key=<DUMMY_KEY>
+      ```
+
+      Click the `Execute` button in the Prometheus console again until a count other than `0` is returned
+
+      Now, every few seconds click the `Execute` button. As soon as the count goes back to `0` run the command prepared earlier in this section. This ensures that we send our requests in alignment with the Prometheus metrics scrape interval:
+
+      ```sh
+      for i in {1..$numRequests}; do curl -i https://<DUMMY_URL>//?user_key=<DUMMY_KEY>; done
```

Comment:
With the script above, I wasn't able to reach the alert thresholds, because those curl commands took too long to finish. If I ran the commands in background, they were working fine.
```suggestion
      for i in {1..$numRequests}; do curl -i https://<DUMMY_URL>//?user_key=<DUMMY_KEY>& done
```